SAMv1.tex

```diff
@@ -337,6 +340,24 @@ \subsection{The header section}
 \end{longtable}
 \end{center}
 
+\subsubsection{Defined sub-sort terms}\label{sec:sub-sort}
+
+While {\tt SS} sub-sort field allows implementation-defined keywords, some terms are predefined with specific meanings.
+
+\begin{description}
+\item[lexicographical] sort order is defined as a character-based dictionary sort with the character order as defined by the POSIX C locale.
+  For example ``abc'', ``abc17'', ``abc5'', ``abc59'' and ``abcd'' are in lexicographical order.
+
+\item[natural] sort order is similar to lexicographical order except that runs of adjacent digits are considered to be numbers embedded within the text string, ordered numerically when compared to each other and ordered as single digits when compared to the surrounding non-digit characters.
+Runs that differ only in the number of leading zeros (thus are numerically tied) are ordered by more-zeros coming before fewer-zeros.
+The characters `\texttt{-}' and `\texttt{.}' are considered as ordinary characters, so apparently negative or fractional values are not treated as part of an embedded number.
+  For example, ``abc'', ``abc+5'', ``abc-5'', ``abc.d'', ``abc03'', ``abc5'', ``abc008'', ``abc08'', ``abc8'', ``abc17'', ``abc17.+'', ``abc17.2'', ``abc17.d'', ``abc59'' and ``abcd'' are in natural order.
+
+\item[umi] is a lexicographical sort by the UMI tag.
```

Comment:
How late is it to argue that this should be called `molecule`, so we don't have acronyms as keywords?  A UMI is a unique _molecular_ identifier.

Reply:
This was a suggested addition by @yfarjoun I believe.  It's not something I've seen or used before.  Yossi care to comment?

I'm happy with either umi or molecule for what it's worth, although would note perhaps umi is more accurate as we'll typically be sorted by the *identifier* which may contain non sequence bases.

Reply:
Makes sense; for my comment, take it or leave it.